src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Try to fix all eslint rules and remove comments

Reply:
Got it, thanks!